src/backend/utils/fmgr/funcapi.c

```diff
@@ -170,6 +185,15 @@ shutdown_MultiFuncCall(Datum arg)
 	FmgrInfo   *flinfo = (FmgrInfo *) DatumGetPointer(arg);
 	FuncCallContext *funcctx = (FuncCallContext *) flinfo->fn_extra;
 
+	/*
+	 * Updated SRF call protocol might have SRF never called, that's why
```

Comment:
I think "Updated" won't make sense outside the context of this patch. I think we can just leave the fact that there may not be a context if the SRF function has never been called.